docs/reference/method/MongoDBClient-startSession.txt

```diff
@@ -2,10 +2,10 @@
 MongoDB\\Client::startSession()
 ===============================
 
-.. default-domain:: mongodb
-
 .. versionadded:: 1.3
 
+.. default-domain:: mongodb
```

Comment:
This rendered fine before. Just re-ordering it for consistency with other RST files.